jupyter-opengl/Dockerfile

```diff
@@ -24,10 +22,8 @@ RUN apt-get update && \
     xvfb && \
     rm -rf /var/lib/apt/lists/*
 
-COPY requirements.yml .
-# Downgrade ipywidgets to less than 8 to avoid https://github.com/matplotlib/ipympl/issues/460
-RUN pip install --no-cache-dir --ignore-installed "ipywidgets>=7.0,<8"
-RUN mamba env update --file requirements.yml && rm requirements.yml
+# COPY requirements.yml .
+# RUN mamba env update --file requirements.yml && rm requirements.yml
```

Comment:
Do we still need these two lines as they are commented out?

Reply:
I've rebased this branch to pick up the changes from the previous branch :+1: